sdk/go/common/workspace/project.go

```diff
@@ -122,7 +135,7 @@ type Project struct {
 	License *string `json:"license,omitempty" yaml:"license,omitempty"`
 
 	// Config has been renamed to StackConfigDir.
-	Config interface{} `json:"config,omitempty" yaml:"config,omitempty"`
+	Config map[string]ProjectConfigType `json:"config,omitempty" yaml:"config,omitempty"`
```

Comment:
I've implemented project rewriting such that when we validate/marshal the project, if config was a string, it would already have been moved to `stackConfigDir` ✅